apps/web/src/pages/Home/IAPHomePage/Home.tsx

```diff
@@ -490,17 +535,28 @@ export const Home = ({ latestPool }: HomeProps) => {
           backgroundSize: "cover",
           backgroundPosition: "top center",
         }}
+        role="img"
```

Comment:
Bummer, Axe doesn't like this.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/e177c921-9cd6-4c4c-8816-a3a6a4dacff6)

Reply:
Yeah, we can't do this 😅 I alsol beleive if an image is important enough to provide alt text, it should not be used as an obscured background image.

Though, if we need the text, you can set the container to `position: relative` and so some funny absolute posiiton on an img to get it to sit in behind.

```css
.wrapper { position: relative; }

.content {
  position: relative;
  z-index: 2;
}

.img {
  position: absolute;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  object-fit: cover;
  z-index: 1;
}
```

Reply:
Gray OK'd not alt-texting these.
[rollback div tagging and quotes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8033/commits/0b070e907fb0cfe35fe61bc0e7b05dbdde3476c3)